spec.md

```diff
@@ -1544,6 +1549,11 @@ delete any resources associated with the Service Binding. In the case where
 credentials were generated, this might result in requests to the Service
 Instance failing to authenticate.
 
+If a Service Broker accepts the request to delete a Service Binding during
+the process of it being created, then it MUST have the net effect of halting
+the current creation process and deleting any resources associated with
+the Service Binding.
```

Comment:
Does this imply that the broker MUST consistently return consistent `410 Gone` responses to the `Polling Last Operation for Service Bindings` endpoint calls (i.e. prevent race conditions among binding creation and binding deletion)? Instead could a compliant broker be allowed to serialize delete binding execution after the binding creation completes ? 

In other words, should a client `Polling Last Operation for Service Bindings` endpoint potentially expect to receive a `200 OK` and then when polled again receive a `410 Gone` response?

Reply:
hmmm great question. I think a 410 might be appropriate but it might also really confuse existing brokers. I wonder if we should require the last_op to return a 200 + a status of "failed", and then the "description" can include something like "failed due to a delete request".  Let me try that and so you guys can see how it looks.